requests-handler.ts

```diff
@@ -0,0 +1,133 @@
+import {DefaultContext, ExtendableContext} from 'koa';
+import {
+    retrieveAllProducts,
+    retrieveSpecificProduct,
+    createProduct,
+    updateProduct,
+    deleteProduct,
+    checkout
+} from './db-actions';
+import {IRouterParamContext} from 'koa-router';
+import {creationSchema, updateSchema, checkoutSchema} from './request-schemas';
+import {logger} from './logger';
+
+const sendAllProductsRequest = async (ctx: ExtendableContext & IRouterParamContext) => {
```

Comment:
send to whom? its more like `getAllProducts`

Reply:
done